ui/packages/catalog-ui-search/src/main/webapp/component/query-settings/query-settings.hbs

```diff
@@ -34,14 +37,16 @@
             Edit
         </span>
     </button><!--
-    --><button class="editor-save is-positive">
+    -->
+    <button class="editor-save is-positive">
```

Comment:
These were intended to be how they were

Reply:
I will revert, and also leave a comment explaining why.